tests/ci/codebuild/linux-x86/minimal_tests/aws-crt-manylinux2014-x64_gcc-4x.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+phases:
+  install:
+    on-failure: ABORT
+    commands:
+      # Install gcc-4x.
+      # aws-crt-manylinux2014 has gcc-9 installed. aws-lc compiler test cannot work with the compiler.
```

Comment:
Instead of compiling with a different compiler should we use this as an opportunity to pass in the `BUILD_TESTING=0` `BUILD_SSL=0` to test the build without tests/C++ and replicate there build with the same config?

Reply:
Added.

Reply:
This dimension was tested in my personal account and passed.